더미 데이터 생성 코드 작성

June222 opened the pull request:
아래의 코드로 더미 데이터를 작성할 수 있음. #8 
```bash
python manage.py seed testapp --number=15

python manage.py seed_users --user=10 --study=5
```

---

models.py 에서 더미 데이터 생성 시 오류 만드는 부분이 있어 해결함 #9 

1. related_name  추가
```python
leader_id = models.OneToOneField(User, on_delete=models.DO_NOTHING, related_name="leader", null=True)
users = models.ManyToManyField(User, related_name="participants")
```
2. SET_NULL 변경
```python
writer_id = models.ForeignKey(User, on_delete=models.SET_NULL)
writer_id = models.ForeignKey(User, on_delete=models.DO_NOTHING)
```

June222 committed:
docs: add how to use django-seed.md

- django-seed를 이용하여 dummy data를 작성하는 2가지 방법에 대해 작성.

- dummy 데이터가 필요한 경우 빠른 환경설정을 위해 문서를 작성.

issue #8

June222 committed:
chore: add files to use django-seed(dummy data)

docs.dummy_data_docs.md 참조

June222 committed:
feat: add seed_users.py

- user, study에 대한 더미데이터 생성 조건 파일 작성

- `python manage.py seed_users --user=10 --study=5` 로 사용할 수 있음 docs 참조

issue #8, #9

sihyeong671 commented:
Do nothing보다 null=True로 바꾸는게 좋지 않을까요?

June222 committed:
Merge branch 'dev' of https://github.com/PNU-AID/AID_Backend_v2 into dev

June222 committed:
chore: django-seed 삭제 및 mysql로 DB 변경

django-seed는 구체적인 조건을 작성하기 힘들어서 삭제한 것 같음.

June222 committed:
fix: do_nothing is not good code

do_nothing이라고 하지말고, null=true로 바꿔서 set_null 유지.

sihyeong671 commented:
개발시에는 migrations 폴더내의 파일은 commit 빼주세요
gitignore에 추가해놨는데 `git pull upstream dev` 하고 다시 PR 부탁드립니다.

June222 committed:
refactor: remove _id from foreign key field name

- foreign key로 설정한 field의 column name은 자동적으로 _id가 추가됨.
그래서 question table의 경우, writer_id가 실제
테이블에는 writer_id_id로 col name 이 설정됨.

- 따라서 _id를 삭제
